app/mm-redux/client/client4.ts

```diff
@@ -3043,6 +3044,26 @@ export default class Client4 {
         );
     };
 
+    // Apps
+
+    getAppsProxyRoute() {
+        return `${this.url}/plugins/com.mattermost.apps`;
+    }
+
+    executeAppCall = async (call: AppCall) => {
+        return this.doFetch(
+            `${this.getAppsProxyRoute()}/api/v1/call`,
+            {method: 'post', body: JSON.stringify(call)},
+        );
+    }
+
+    getAppsBindings = async (userID: string, channelID: string) => {
+        return this.doFetch(
```

Comment:
Can we file a ticket about the (http) cache-ability of this request?

Reply:
A different one to this one? https://mattermost.atlassian.net/browse/MM-31123